app/views/content/train-to-be-a-teacher/initial-teacher-training.md

```diff
@@ -7,12 +7,13 @@ description: |-
 navigation: 20.15
 navigation_title: What to expect in teacher training
 navigation_description: Find out what your teacher training will be like, what you'll learn, and what classroom experience you'll get.
+cta_mailinglist:
+  mailinglist:
+    text: Discover how you can apply for teacher training, including top tips on writing a successful application.
```

Comment:
Again, I know this is in prod already but is there a better way of saying this rather than `Discover`?